src/main/java/com/dougnoel/sentinel/steps/TableSteps.java

```diff
@@ -145,4 +148,23 @@ public static void storeTableValues(int pageNumber, String tableName) {
     public static void clickColumnHeaderToSort(String columnName, String tableName) {
     	getElementAsTable(tableName).clickColumnHeader(columnName);
     }
+
+	/**
+	 * Clicks the xpath of element in row which have stored value in column name
+	 * <p>
+	 * <b>Gherkin Examples:</b>
+	 * <ul>
+	 * <li>I find row in the Project Files Table with value entered for the Search Input and click the xpath //i[contains(concat(' ', @class, ' '), ' fa-toggle-on ')]</li>
+	 * </ul>
+	 * @param tableName String the name of the table element
+	 * @param key String the key used to retrieve the value
+	 * @param xpath String xpath value to element to click
+	 */
+	@When("^I find row in the (.*?) with value (?:entered|selected|used) for the (.*?) and click the xpath (.*?)$")
+	public static void clickStoredTextRowContainsXpath(String tableName, String key, String xpath) {
```

Comment:
Is this functionality not already covered by `TableSteps.clickAssociatedLinkInTable`? How is this different?